xhr.bs

```diff
@@ -1048,9 +1048,11 @@ method must run these steps:
  <li><p><a>Fire an event</a> named <a event><code>readystatechange</code></a>.
 
  <li><p><a>Fire a progress event</a> named <a event><code>load</code></a>
+ at [=this=]
  with <var>transmitted</var> and <var>length</var>.
 
  <li><p><a>Fire a progress event</a> named <a event><code>loadend</code></a>
+ at [=this=]'s {{XMLHttpRequestUpload}} object
```

Comment:
The intended target isn't clear in context. Firefox, Chrome, Edge, and Safari all fire this event at the request upload object.